### PR TITLE
Better inline style matching regex and fix for visibility declaration when contained no spaces

### PR DESCRIPTION
**Pull request by ryanturner10**

### CODE REVIEW

**Comment by jlmakes** [comment]
Nice!

I understand the spirit of your changes, but I'll have to spend a little time deconstructing this regex:

```regex
/([\w|:|\-\s|\.]+)/ig
```

Also, great catch on case insensitivity.

Thanks @ryanturner10

File: dist/scrollreveal.es.js [hunk 1]
```diff
@@ -667,13 +667,13 @@ function style (element) {
 	var inline = {};
 
 	var inlineStyle = element.node.getAttribute('style') || '';
-	var inlineMatch = inlineStyle.match(/.+[^;]/g);
+	var inlineMatch = inlineStyle.match(/([\w|:|\-\s|\.]+)/ig) || [];
```
Comment (jlmakes) [inline dist/scrollreveal.es.js]: I like this, ensuring `inlineMatch` is an array with or without a match.

File: src/instance/functions/style.js [hunk 2]
```diff
@@ -13,13 +13,13 @@ export default function style (element) {
 	const inline = {}
 
 	const inlineStyle = element.node.getAttribute('style') || ''
-	const inlineMatch = inlineStyle.match(/.+[^;]/g)
+	const inlineMatch = inlineStyle.match(/([\w|:|\-\s|\.]+)/ig) || []
```
Comment (jlmakes) [inline src/instance/functions/style.js]: I like this, ensuring `inlineMatch` is an array with or without a match.

Comment (jlmakes) [reply src/instance/functions/style.js]: So for example, `-webkit-transform: rotate(30deg)` is not matched by that regex.

Using the wonderful [RegExr](www.regexr.com), I was able to piece together this one that seems to match inline CSS quite well:

```regex
/[\w-]+\s*:\s*[^;]+\s*/ig
```

I appreciate the contribution though, the case insensitivity alone was a great fix.

**Comment by jlmakes** [comment]
I was inspired by your changes, and came up with this:

```js
const inline = {}
const inlineStyle = element.node.getAttribute('style') || ''
const inlineMatch = inlineStyle.match(/[\w-]+\s*:\s*[^;]+\s*/gi) || []

inline.computed = inlineMatch ? inlineMatch.map(m => m.trim()).join('; ') + ';' : ''

inline.generated = inlineMatch.some(m => m.match(/visibility\:\s?visible/i))
	? inline.computed
	: [...inlineMatch, 'visibility: visible'].map(m => m.trim()).join('; ') + ';'
```

The case insensitivity alone was a great fix, thanks @ryanturner10

File: src/instance/functions/style.js (hunk 2 above)
Comment (ryanturner10) [reply src/instance/functions/style.js]: Ahh good stuff mate, yeah your regex does seem a lot more solid than mine!

**Pull request closed without merging**